ffi/src/version.rs

```diff
@@ -0,0 +1,17 @@
+use std::ffi::{c_char, CStr};
+
+const VERSION: &str = concat!(env!("CARGO_PKG_VERSION"), "\0");
```

Comment:
does this work if the environment variable is not set?

Reply:
No, it will fail to compile if the variable is not set. Check https://doc.rust-lang.org/std/macro.env.html:
> If the environment variable is not defined, then a compilation error will be emitted.

However, it is set by `cargo` during build.